src/content/docs/nrql/using-nrql/funnels-evaluate-data-series-related-events.mdx

```diff
@@ -116,9 +112,9 @@ Here's a detailed breakdown of technical details and constraints for funnel quer
   </tbody>
 </table>
 
-## Funnel query example [#example]
+## Funnel example [#example]
 
-This example queries the <InlinePopover type="browser" /> [`PageView`](/attribute-dictionary/?event=PageView) event and its attributes. It queries unique browser sessions that have progressed from browsing a product, to adding to cart, to checkout. Labels are included for each step, indicated by the keyword `AS`.
+This example queries the <InlinePopover type="browser" /> [`PageView`](/attribute-dictionary/?event=PageView) event and its attributes. It queries unique browser sessions that have progressed from browsing for products, to adding them to their cart, to checking out. It includes labels for each step, indicated by the keyword `AS`.
```

Comment:
Is there a reason humanize this? It's people who are doing these things.

"It queries unique browser sessions from your users as they progress through browsing for products, etc etc"